packages/front-end/components/Features/FeatureModal/index.tsx

```diff
@@ -171,7 +173,11 @@ export default function FeatureModal({
   let ctaEnabled = true;
   let disabledMessage: string | undefined;
 
-  if (!permissions.check("createFeatureDrafts", project)) {
+  if (
+    !permissionsUtil.canManageFeatureDrafts({
+      project: featureToDuplicate?.project || project,
+    })
+  ) {
```

Comment:
Similar to the bug above - if the user's global role was `noaccess`, but they had multiple project-specific roles, and they tried duplicating a feature in a project they have permission to create a feature in, it was using the current project (none), and thus checking the user's global role

Reply:
I think we want `?? project` instead of `|| project` to match the logic used inside of FeatureModal when duplicating.